packages/thumbnail-grid-block/src/ThumbnailGridBlock.tsx

```diff
@@ -0,0 +1,211 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { FC, useEffect, useState } from 'react';
+import '@frontify/fondue-tokens/styles';
+import { BlockProps } from '@frontify/guideline-blocks-settings';
+import { DndContext, DragEndEvent, closestCenter } from '@dnd-kit/core';
+import { SortableContext } from '@dnd-kit/sortable';
+import { restrictToFirstScrollableAncestor } from '@dnd-kit/modifiers';
+import {
+    Asset,
+    AssetChooserObjectType,
+    FileExtensionSets,
+    useAssetUpload,
+    useBlockAssets,
+    useBlockSettings,
+    useEditorState,
+    useFileInput,
+} from '@frontify/app-bridge';
+import { generateRandomId } from '@frontify/fondue';
+import 'tailwindcss/tailwind.css';
+import { Settings, Thumbnail } from './types';
+import { useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+import { ThumbnailImagePlaceholder, ThumbnailItem } from './components/Thumbnail';
+import { gutterPresetMap } from './helper/mapper';
+import { Grid } from './components/Grid/Grid';
+import { ThumbnailRichTextEditors } from './components/Thumbnail/ThumbnailRichTextEditors';
+import { ThumbnailImageWrapper } from './components/Thumbnail/ThumbnailImageWrapper';
+import { getThumbnailStyles } from './helper/StylingHelper';
+
+export const ThumbnailGridBlock: FC<BlockProps> = ({ appBridge }) => {
```

Comment:
```suggestion
export const ThumbnailGridBlock = ({ appBridge }: BlockProps) => {
```
remove also the `FC` import